node-runner-cli/templates/systemd-default.config.j2

```diff
@@ -22,11 +22,14 @@ network.host_ip={{common_config.host_ip}}
 
 log.level=debug
 
-api.port=3334
+api.port={{core_node.core_api_port}}
```

Comment:
so do I correctly understand that `3334` (that was here before) was wrong?
(because you now effectively change it to default `3333`)

Reply:
Indeed. The port was not the default. This brings it back to use the default.